extra/import_data.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import os
```

Comment:
I used flake8 to verify the format of this script.
Could you go over these warnings and resolve them? 

./extra/import_data.py:3:1: F401 'os' imported but unused
./extra/import_data.py:6:1: F401 'sys' imported but unused
./extra/import_data.py:36:128: E501 line too long (132 > 127 characters)
./extra/import_data.py:38:128: E501 line too long (131 > 127 characters)
./extra/import_data.py:45:128: E501 line too long (156 > 127 characters)
./extra/import_data.py:54:17: F841 local variable 'exception' is assigned to but never used

Reply:
the sys and os are correct (and also visible in Visual Studio Code. overlooked and will remove them
the "exception" I knew about and I want this to stay for quick printing of Alissa Errors if needed
the lines are too long I reformatted them back after using autopep8. I thought the original was better for reading